libs/island-ui/core/src/lib/DatePicker/DatePicker.tsx

```diff
@@ -140,50 +124,118 @@ export const DatePicker: React.FC<DatePickerProps> = ({
             setDatePickerState('closed')
             handleCloseCalendar && handleCloseCalendar(startDate)
           }}
-          onChange={(date: Date) => {
+          onChange={(date: any) => {
```

Comment:
👾  why?

Reply:
lets just say this is still a draft 😄  good spot by the way!